_pages/policies/travel-and-leave-policies/leave.md

```diff
@@ -69,18 +69,27 @@ Employees must report to work or have leave approved every scheduled work day. L
 
 Should an unforeseen emergency or illness arise requiring the use of leave not previously approved, contact your supervisor no later than two hours after the start of your typical work day to request permission for the absence. Unless other arrangements were made, requests for annual or sick leave must be made on each day of absence. Failure to properly request leave in accordance with this procedure may result in the denial of leave and a charge of absence without leave (AWOL), and may be the basis for disciplinary action. Absence without leave is an absence from duty that is not authorized or approved.
 
+### Leave checklist
 Before taking leave, be sure to:
 
 - Submit leave through [HR Links](https://corporateapps.gsa.gov/hr-links/)
-- Notify your Engagement Manager, Account Manager, or project lead
-- Mark your out of office time on your calendar
+- Notify your supervisor of the dates. Also notify your Engagement Manager, Account Manager, and project lead if applicable.
+- Mark your out of office time on your calendar and any applicable team or project calendars. Remember to set the permissions of the event to "public" so that people can see it's OOO time, rather than just seeing "busy."
 - Tell team members or partners via Slack or email, and make sure work is covered
-- Set up an email autoresponder as appropriate
-- And, for 18F team members:
-    - Submit [correct Tock entries](#time-tracking)
-    - Update your Slack username and profile (ex: `@username, OOO through 6/27`)
-    - Update the [18F Out of Office Calendar](https://calendar.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com)
-    - For a more detailed checklist, see [18F Out of Office expectations](https://docs.google.com/document/d/1qgA-vEQ1_t5plPCAs1aCOAjK0RXQzka2hHSVhoa6v5o/edit)
```

Comment:
Are we just recreating the OOO expectations here?

Reply:
Yes, plus a few additional benefits:
1) The OOO expectations doc is incomplete - Product chapter members have a requirement that isn't reflected therein
2) It's more of a merge than recreation - some of the checklist items were duplicated between the handbook and the Google Doc, so we get to effectively eliminate some content by moving it all into the handbook.
3) The portion of the OOO expectations that isn't related to leave (what to do when working from remote locations) is now referenced in the relevant parts of the handbook so hopefully it's found a little more easily.

Reply:
Cool. I had assumed we maintained the separate doc because of some sensitive info we couldn't put in the handbook. If that's not the case and we can retire a google doc, even better.